build/common.props

```diff
@@ -2,12 +2,13 @@
   <PropertyGroup>
     <!-- Extensions can have independent versions and only increment when released -->
     <Version>3.0.0$(VersionSuffix)</Version>
-    <ExtensionsVersion>4.0.4$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
-    <CosmosDBVersion>4.0.0$(VersionSuffix)</CosmosDBVersion>
-    <HttpVersion>3.1.1$(VersionSuffix)</HttpVersion>
-    <MobileAppsVersion>3.0.0$(VersionSuffix)</MobileAppsVersion>
-    <SendGridVersion>3.0.2$(VersionSuffix)</SendGridVersion>
-    <TwilioVersion>3.0.1$(VersionSuffix)</TwilioVersion>
+    <ExtensionsVersion>5.0.0-beta.1$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
+    <CosmosDBVersion>4.0.1$(VersionSuffix)</CosmosDBVersion>
```

Comment:
I don't think we need to rev the versions of all the other packages - they're not changing at all and don't have a proj dependency on Extensions, right?